specifyweb/workbench/upload/clone.py

```diff
@@ -97,7 +97,7 @@ def _cloned(value, field, is_dependent):
             ).all()  # Clone all records separatetly
         ]
         for (field, is_dependent) in marked
-        if is_dependent and not field.concrete
+        if is_dependent and not field.concrete and not (field.one_to_one or field.many_to_one)
```

Comment:
Had to add this because it tried to clone `childCO` and `childCog` records which are one-to-ones and would throw an error